Makefile

```diff
@@ -25,6 +25,18 @@ deploy: manifests
 	kubectl apply -f config/crds
 	kustomize build config/default | kubectl apply -f -
 
+deploy-dev: manifests
+	kubectl apply -f config/crds
+	kustomize build config/overlays/development | kubectl apply -f -
+
+undeploy: manifests
```

Comment:
Should we call this clean?

Reply:
Happy to - just wanted to match the auto-generated deploy targets.

Reply:
is this folder structure working for you guys... I get this sad error: 
Error: rawResources failed to read Resources: Load from path ../rbac/rbac_role.yaml failed: security; file '../rbac/rbac_role.yaml' is not in or below '/home/rakelkar/go/src/github.com/kubeflow/kfserving/config/default' 

You're possibly using an older version of kustomize?
https://github.com/kubernetes-sigs/kustomize/issues/766

Reply:
I think this is an issue with kubebuilder which is fixed in the unreleased master : kubernetes-sigs/kubebuilder/issues/654

Reply:
I think we need to decide to staywith current kubebuilder release v1.0.8 and kustomize 1.x and then move to kustomize 2.x when kubebuilder releases what is in HEAD?